js/components/Banner/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Image } from 'react-native';
+import styles from './styles';
+
+const Banner = () => {
+
+  const bgPath = require('./../../../assets/images/bg.jpg');
+  const logoPath = require('./../../../assets/images/logo_shadow.png');
```

Comment:
What I meant was making a variable for the path, then using it to build the full path.

```js
const assetsPath = './../../../assets/images');
```

Then you can use:

```js
require(`${assetsPath}/bg.jpg`)
```